mempool/src/error.rs

```diff
@@ -81,6 +82,22 @@ pub enum TxValidationError {
     AncestorFeeUpdateOverflow,
     #[error("Transaction is a descendant of expired transaction.")]
     DescendantOfExpiredTransaction,
+    #[error("Chainstate error")]
+    ChainstateError,
+    #[error("Subsystem call error")]
+    CallError,
     #[error("Internal Error.")]
     InternalError,
 }
+
+impl From<subsystem::subsystem::CallError> for TxValidationError {
```

Comment:
You don't need to do these manually. See how the `#[from]` macro is used in other places to automatically do these for you.

Reply:
Thanks for reminding me of that!